Tasks.cs

```diff
@@ -0,0 +1,302 @@
+﻿using DICUI.Data;
+using DICUI.Utilities;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace DICUI
+{
+    class Result
```

Comment:
IIRC, default visibility of classes is `internal`. It's not an official thing, but I tend to like to explicitly put access modifiers on everything. In this case, I'd call it `public`. Not necessary, but fun.